driver-core/src/main/com/mongodb/internal/client/model/FindOptions.java

```diff
@@ -511,6 +512,32 @@ public FindOptions showRecordId(final boolean showRecordId) {
         return this;
     }
 
+    /**
+     * Returns the allowDiskUse value
+     *
+     * @return the allowDiskUse value
+     * @since 4.0
```

Comment:
Don't need this since this class is now internal

Reply:
Done